src/domain/bitcoin/onchain/index.types.d.ts

```diff
@@ -65,3 +71,63 @@ interface IOnChainService {
     targetConfirmations: TargetConfirmations,
   ): Promise<Satoshis | OnChainServiceError>
 }
+
+interface IBitcoindService {
```

Comment:
I need to think about the boundaries of these interfaces some more. I don't like that `Bitcoind` is in the name.
I think there can be a lot smaller more granular interfaces like `IBlockExplorer` and `IOnChainWalletRepository` or something like that